VarUsage does not appear to log the line number the bad var usage is found on

As in title. Sample compile error message:

```
/home/james/git/loom-experiments/loom-experiments-core/src/main/java/io/jbaker/loom/raft/Raft.java:308: warning: [UnusedMethod] Method 'runOneIteration' is never used.
        Duration runOneIteration();
                 ^
    (see https://errorprone.info/bugpattern/UnusedMethod)
  Did you mean to remove this line?
/home/james/git/loom-experiments/loom-experiments-core/src/main/java/io/jbaker/loom/raft/Raft.java:321: warning: [UnusedMethod] Constructor 'ServerImpl' is never used.
        private ServerImpl(
                ^
    (see https://errorprone.info/bugpattern/UnusedMethod)
/home/james/git/loom-experiments/loom-experiments-core/src/main/java/io/jbaker/loom/raft/Raft.java:311: warning: [UnusedNestedClass] This nested class is unused, and can be removed.
    private static final class ServerImpl implements Server, BackgroundTask {
                         ^
    (see https://errorprone.info/bugpattern/UnusedNestedClass)
  Did you mean to remove this line?
error: [VarUsage] The `var` keyword results in illegible code in most cases and should not be used.
    (see https://github.com/palantir/gradle-baseline#baseline-error-prone-checks)
  Did you mean 'for (ListenableFuture<AppendEntriesResponse> future : otherServers.values().stream()'?
1 error
3 warnings

```

Obviously I can just find the matching code, but I suspect there's a small mistake in the definition of the check.

--- a/baseline-error-prone/src/main/java/com/palantir/baseline/errorprone/VarUsage.java
+++ b/baseline-error-prone/src/main/java/com/palantir/baseline/errorprone/VarUsage.java
@@ -64,7 +64,7 @@ public final class VarUsage extends BugChecker implements BugChecker.VariableTre
                     && token.hasName()
                     && token.name().contentEquals("var")) {
                 SuggestedFix.Builder fix = SuggestedFix.builder();
-                return buildDescription(typeTree)
+                return buildDescription(tree)
                         .addFix(fix.replace(
                                         token.pos(),
                                         token.endPos(),

--- a/changelog/@unreleased/pr-2244.v2.yml
+++ b/changelog/@unreleased/pr-2244.v2.yml
@@ -1,0 +1,5 @@
+type: fix
+fix:
+  description: VarUsage failures reference a real tree
+  links:
+  - https://github.com/palantir/gradle-baseline/pull/2244

